### PR TITLE
[NRLF-243] Deploy swagger

### PR DESCRIPTION
**Pull request by nomad3k**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [x] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/nrl-consumer-api.yaml [hunk 1]
```diff
@@ -1,41 +1,2062 @@
...
+    Use this API to populate an index of a patient’s shared clinical records.
+
+    You can:
+    - manage pointers to clinical records
+
+    You cannot currently use this API to:
+    - access pointers controlled by other producers
+
+    ### Data availability, timing and quality
+    TBC
```
Comment (Rohoolio) [inline specification/nrl-consumer-api.yaml:18]: Are we happy to have this TBC for now?

File: specification/nrl-consumer-api.yaml [hunk 2]
```diff
@@ -1,41 +1,2062 @@
...
+    This access mode is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
+
+    To use this access mode, use the following security pattern:
+    * [Application-restricted RESTful API - signed JWT authentication](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-signed-jwt-authentication)
+
     ## Environments and testing
-    To be completed.
+    | Environment       | Base URL                                                               |
+    | ----------------- | ---------------------------------------------------------------------- |
+    | Sandbox           | `https://sandbox.api.service.nhs.uk/nrl-producer-api/FHIR/R4/`     |
```
Comment (Rohoolio) [inline specification/nrl-consumer-api.yaml:79]: Can we use the real domain name for this yet or do we want to wait till we know the paths for sure? eg api.record-locator.national.nhs.uk

File: specification/nrl-consumer-api.yaml [hunk 3]
```diff
@@ -1,41 +1,2062 @@
...
+    This API is [in development](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses).
+
+    To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=immunisation-history-api).
+
+    If you have any other queries, [contact us](https://digital.nhs.uk/developer/help-and-support).
+
     ## Service level
-    To be completed.
+
+    This API is a TBC service, meaning it is operational and supported TBC.
```
Comment (brianbayes) [inline specification/nrl-consumer-api.yaml:40]: Are we ok with leaving TBC here?

File: specification/nrl-consumer-api.yaml [hunk 4]
```diff
@@ -1,41 +1,2062 @@
...
+    Alternatively, you can try out the sandbox using our Postman collection:
+
+    [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/TBC)
+
+    ### Integration testing
+    Our [integration test environment](https://digital.nhs.uk/developer/guides-and-documentation/testing#integration-testing):
+    * is for formal integration testing
+    * includes authorisation
+
+    It also includes ready-to-use TBC test data
```
Comment (brianbayes) [inline specification/nrl-consumer-api.yaml:100]: Are we ok with leaving TBC here?

File: specification/nrl-consumer-api.yaml (hunk 2 above)
Comment (nomad3k) [reply specification/nrl-consumer-api.yaml:79]: Still working on this, so can't correct it yet.

**Review by brianbayes — APPROVED** [review]

**Pull request merged**